airbyte-commons/src/main/java/io/airbyte/commons/json/Jsons.java

```diff
@@ -26,9 +30,14 @@
 import java.util.Set;
 import java.util.function.BiConsumer;
 import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class Jsons {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(Jsons.class);
+  public static final String SECRET_MASK = "******";
```

Comment:
I think this functionality does not belong in this class. `Jsons` should be a very generic class containing only common JSON helpers and no business logic, but this mask is very coupled with oauth business-logic